tests/rptest/tests/redpanda_test.py

```diff
@@ -55,13 +56,15 @@ def __init__(self,
         if self.si_settings:
             self.si_settings.load_context(self.logger, test_context)
 
-        self.redpanda = RedpandaService(test_context,
-                                        num_brokers,
-                                        extra_rp_conf=extra_rp_conf,
-                                        enable_pp=enable_pp,
-                                        enable_sr=enable_sr,
-                                        si_settings=self.si_settings,
-                                        **kwargs)
+        self.redpanda = RedpandaService(
+            test_context,
+            num_brokers,
+            extra_rp_conf=extra_rp_conf,
+            enable_pp=enable_pp,
+            enable_sr=enable_sr,
+            si_settings=self.si_settings,
+            node_ready_timeout_s=node_ready_timeout_s,
+            **kwargs)
```

Comment:
correct me if i'm wrong, but you don't need to pass node_ready_timeout_s explicitly. it will already be passed through via kwargs.

Reply:
can't do it cuz you're right